contributors/devel/development.md

```diff
@@ -286,7 +286,7 @@ cd $working_dir/kubernetes
 make test
 
 # Run package tests verbosely
-make test WHAT=pkg/util/cache GOFLAGS=-v
+make test WHAT=k8s.io/kubernetes/pkg/util GOFLAGS=-v
```

Comment:
As there is no code/test files under `pkg/util` now (removed by https://github.com/kubernetes/kubernetes/pull/48256), this example should be changed accordingly.